### PR TITLE
Scrum 1491 2

### PR DESCRIPTION
**Pull request by ianlongden**
Ready for PR.

### CODE REVIEW

**Comment by ianlongden** [comment]
authors, cross_references, editors, mesh_details and mod_reference_types all had table names made singular.  Still to do references, resources and users.

File: backend/app/literature/crud/reference_crud.py [hunk 1]
```diff
@@ -49,30 +49,37 @@ def create(db: Session, reference: ReferenceSchemaPost): # noqa
     :return:
     """
```
Comment (azurebrd) [inline backend/app/literature/crud/reference_crud.py:51]: There's still a print statement here

File: backend/app/literature/crud/reference_crud.py [hunk 2]
```diff
@@ -49,30 +49,37 @@ def create(db: Session, reference: ReferenceSchemaPost): # noqa
...
 
+    logger.debug("creating reference")
+    print(reference)
```
Comment (azurebrd) [inline backend/app/literature/crud/reference_crud.py:53]: There's still a print statement here

File: backend/app/literature/crud/reference_crud.py [hunk 3]
```diff
@@ -113,12 +123,17 @@ def create(db: Session, reference: ReferenceSchemaPost): # noqa
...
 
+    logger.debug("add reference")
     reference_db_obj = ReferenceModel(**reference_data)
+    logger.debug("have model, save to db")
     db.add(reference_db_obj)
+    logger.debug("saved")
     db.commit()
 
     for field, value in vars(reference).items():
+        logger.debug("Porcessing mod corpus asso")
```
Comment (azurebrd) [inline backend/app/literature/crud/reference_crud.py:136]: typos, but doesn't matter

File: src/xml_processing/tests/pmids_by_mods [hunk 4]
```diff
@@ -1,5 +1,5 @@
 10206683	1	RGD
-10022914	3	RGD, FB, WB
```
Comment (azurebrd) [inline src/xml_processing/tests/pmids_by_mods:2]: I think this file shouldn't change.  It gets changed by the functional tests, but the changes shouldn't get committed.  Maybe we should add to the .gitignore ?

File: src/xml_processing/tests/inputs/alliance_pmids [hunk 5]
```diff
@@ -1,6 +1,6 @@
 2
-10206683
```
Comment (azurebrd) [inline src/xml_processing/tests/inputs/alliance_pmids:2]: This file also shouldn't change, but this seems like it hasn't really changed, just moved an indentifier further down

**Review by azurebrd — CHANGES_REQUESTED** [review]
It looks really good, thanks @ianlongden !  There's a couple of files at the end with comments that they shouldn't change.  I'm pretty sure the first one is okay because the data hasn't changed, but the second one probably shouldn't change.  I could be wrong and it might be okay to have it change.  If you've tested that without the change things don't work, then keep the change.  But I think it's a change generated by the functional test that shouldn't get committed.   All the other comments are minor and fine with me.

File: backend/app/literature/crud/reference_crud.py (hunk 2 above)
Comment (ianlongden) [reply backend/app/literature/crud/reference_crud.py]: Done.

**Review by azurebrd — APPROVED** [review]
That's great, thanks @ianlongden

**Pull request merged**